mc-sema/runtime/linux_amd64_callback.c

```diff
@@ -37,22 +37,27 @@ uint64_t __mcsema_get_max_stack() {
     uint64_t start;
     uint64_t end;
     while( fgets(line, sizeof(line), p) != NULL ) {
```

Comment:
I think that would only happen if the format of /proc/pid/maps changed; fgets is guaranteed to read until max chars, EOF or end of line. If it hits EOF, it returns nullptr.